addons/mail/models/mail_thread.py

```diff
@@ -4179,6 +4182,21 @@ def _get_mail_thread_data(self, request_list):
             ])._format_for_chatter()
             res['selfFollower'] = self_follower[0] if len(self_follower) > 0 else None
             res['followers'] = self.message_get_followers()
+
+            subtype_id = self.env['ir.model.data']._xmlid_to_res_id('mail.mt_comment')
+            self_recipient = self.env['mail.followers'].search([
+                ("res_id", "=", self.id),
+                ("res_model", "=", self._name),
+                ('partner_id', '=', self.env.user.partner_id.id),
```

Comment:
No empty newline.
There's really no reason to add it other than confuse reading at a glance because empty newline gives impression it's a split of 2 concerns when this isn't